app/actions/actionTypes.js

```diff
@@ -1,2 +1,5 @@
 export const LOAD_USER_SUCCESS = 'LOAD_USER_SUCCESS';
 export const GET_USER_SUCCESS = 'GET_USER_SUCCESS';
+export const CREATE_USER_SUCCESS = 'CREATE_USER_SUCCESS';
+export const UPDATE_USER_SUCCESS = 'UPDATE_USER_SUCCESS';
+export const DELETE_USER_SUCCESS = 'DELETE_USER_SUCCESS';
```

Comment:
Yes, I got it @AlphaGit. But as I mentioned, this is a react-redux standard as you can see in the official doc: http://redux.js.org/docs/basics/Actions.html to have the actionTypes separated in another file ;)